src/cli.js

```diff
@@ -1,6 +1,20 @@
 import * as jsApi from './node-jspm-jasmine.js';
+import commander from 'commander';
 
-jsApi.runTests({}, function(err) {
-	console.error(err);
-	process.exit(1);
-});
+export function run(args) {
+
+    commander
+        .option('-c, --jasmineConfig <path>')
+        .option('-c, --packageJSON <path>')
```

Comment:
Sure. I like the idea of having "json" as part of the name, but the alternatives (`--package-json-path`, `--pjson-path`) don't seem too great.